packages/x-tailwindcss/src/x-tailwind-plugin/components/suggestion/default.ts

```diff
@@ -14,7 +14,8 @@ export function suggestionDefault(helpers: TailwindHelpers) {
     fontFamily: theme('fontFamily.main'),
     fontWeight: theme('fontWeight.regular'),
 
-    display: 'flex',
+    display: 'grid',
+    gridAutoFlow: 'column',
     justifyContent: 'center',
     alignItems: 'baseline',
     alignContent: 'center',
```

Comment:
Hi @tajespasarela: I think we should remove `flexFlow: 'row nowrap'` property as we are now using the `CSSGrid` system